src/seaport.ts

```diff
@@ -458,40 +465,29 @@ export class OpenSeaPort {
    * @param referrerAddress The optional address that referred the order
    */
   public async createBundleBuyOrder(
-      { tokenIds, tokenAddresses, assets, accountAddress, startAmount, expirationTime = 0, paymentTokenAddress, sellOrder, schemaName, referrerAddress }:
-      { tokenIds?: string[];
-        tokenAddresses?: string[];
-        assets: Asset[];
+      {  assets, quantities, accountAddress, startAmount, expirationTime = 0, paymentTokenAddress, sellOrder, referrerAddress }:
+      { assets: Asset[];
+        quantities?: number[];
         accountAddress: string;
         startAmount: number;
         expirationTime?: number;
         paymentTokenAddress?: string;
         sellOrder?: Order;
-        schemaName?: WyvernSchemaName;
         referrerAddress?: string; }
     ): Promise<Order> {
 
-    if (!assets && tokenIds && tokenAddresses) {
-      onDeprecated("Use `assets` instead of the `tokenIDs` and `tokenAddresses` list")
-      assets = _.zipWith(tokenIds, tokenAddresses, (tokenId, tokenAddress) => {
-        return { tokenAddress, tokenId }
-      })
-    }
-
+    quantities = quantities || assets.map(a => 1)
```

Comment:
"Magic Number"
Might be good to comment here as this could be mysterious

Reply:
Better yet add `quantities` to the js doc comment and say it defaults to 1 like the `createBuyOrder` method below does